File: ABC086A.ml
(* ABC086A - Product *)

let solver = function
  | [ a; b ] -> if a * b mod 2 = 0 then "Even" else "Odd"
  | _ -> failwith "?"

let () =
  read_line () |> String.split_on_char ' ' |> List.map int_of_string |> solver
  |> print_endline

File: ABC086A.hs
{- ABC086A - Product -}

solver :: [Int] -> String
solver xs = case xs of
  [a, b]  | even (a*b) -> "Even"
          | otherwise ->  "Odd"
  _ -> "?"

main :: IO ()
main =
  putStrLn . solver =<< input
  where
    input = map read . words <$> getLine
